Svc/FprimeDeframer/docs/sdd.md

```diff
@@ -0,0 +1,65 @@
+# Svc::FprimeDeframer
+
+The `Svc::FprimeDeframer` component receives F´ frames on its input port, takes off the header and trailer (or "footer"), and passes the encapsulated payload to other components of the system.
+
+## F Prime frame format
+
+```mermaid
+---
+title: "F Prime Frame Format"
+---
+packet-beta
+  0-31: "Start word: 0xDEADBEEF [4 bytes]"
```

Comment:
Technically aren't these intended to be configurable?

Reply:
Protocol should not be configurable, see https://github.com/nasa/fprime/issues/3332